src/ViewModels/DiffContext.cs

```diff
@@ -3,10 +3,12 @@
 using System.Threading.Tasks;
 
 using Avalonia;
+using Avalonia.Media;
 using Avalonia.Media.Imaging;
 using Avalonia.Threading;
 
 using CommunityToolkit.Mvvm.ComponentModel;
+using SourceGit.Models;
```

Comment:
Please never add namespace using declaration for SourceGit.*. Instead, use Models.FileModeDiff directly in your code.

Reply:
Sorry, I'm new to C#, not familiar with the APIs and codestyle

Reply:
You can run `dotnet format` to apply the code-style defined in .editorconfig